pkg/loader/compose/v1v2.go

```diff
@@ -264,6 +264,16 @@ func libComposeToKomposeMapping(composeObject *project.Project) (kobject.Kompose
 		serviceConfig.MemLimit = composeServiceConfig.MemLimit
 		serviceConfig.TmpFs = composeServiceConfig.Tmpfs
 		serviceConfig.StopGracePeriod = composeServiceConfig.StopGracePeriod
+		var groupAdd []int64
```

Comment:
Please add spacing, newline here as well as a newline below (after `serviceConfig.GroupAdd = groupAdd`).

Since this is a range, perhaps it may be better to make this into a separate function? 

Also add a small comment please :)